pkg/reconciler/autoscaling/hpa/hpa_test.go

```diff
@@ -86,6 +86,10 @@ func TestControllerCanReconcile(t *testing.T) {
 	}
 }
 
+func markReconcile(pa *asv1a1.PodAutoscaler) {
```

Comment:
nit: All the other options are located at the bottom of this file, so this should go there too?

also

```suggestion
func markReconciled(pa *asv1a1.PodAutoscaler) {
```